src/renderer/tr_decals.c

```diff
@@ -1024,8 +1024,8 @@ void R_CullDecalProjectors(void)
 			*dp                                           = temp;
 		}
 
-		decalBits |= (1 << numDecalProjectors);
-		numDecalProjectors++;
+		numDecalProjectors = i + 1;
```

Comment:
Its just same as `numDecalProjectors++` but its just shorter one

`decalBits |= (1 << numDecalProjectors);`
Did you checked if we are hitting the limit of MAX_USED_DECAL_PROJECTORS because for me it looks like the culrplit here?
With your change it gets its value from `i` ?